Source/EventFlow.ReadStores.ElasticSearch/IEsReadModelStore.cs

```diff
@@ -0,0 +1,10 @@
+﻿using EventFlow.Aggregates;
+
+namespace EventFlow.ReadStores.ElasticSearch
+{
+    public interface IEsReadModelStore<TAggregate, TReadModel> : IReadModelStore<TAggregate>
```

Comment:
@tabro The interface isn't used anywhere, maybe just remove it. It might be the same case for the MSSQL part